Tools/WinMLRunner/CommandLineArgs.h

```diff
@@ -34,6 +37,9 @@ class CommandLineArgs
     std::wstring m_modelFolderPath;
     std::wstring m_modelPath;
     std::wstring m_imagePath;
+
+	float m_scale;
```

Comment:
This scale member variable needs to be initialized. In line 339 in BindingUtilities.h, you check if m_scale!=1.0
```suggestion
	float m_scale = 1.0;
```